setup.cfg

```diff
@@ -49,7 +49,10 @@ filterwarnings=
     ignore::pytest.PytestUnraisableExceptionWarning
     ignore::RuntimeWarning:desc.compute
     # Ignore division by zero warnings.
+    ignore:numpy.ndarray size changed:RuntimeWarning
```

Comment:
the comments need to be on their own line fyi

Reply:
^ yea it thinks that you are naming a different module otherwise. And I think that this is fine, only we look at it and we can figure out which go with which